brownfield_django/settings_shared.py

```diff
@@ -68,3 +69,18 @@
         'rest_framework.permissions.IsAuthenticated',
     ]
 }
+
+
+MEDIA_URL = "/uploads/"
+MEDIA_ROOT = 'uploads'
+STATIC_URL = "/media/"
+STATIC_ROOT = "/tmp/brownfield_django/static"
+STATICFILES_DIRS = ('media/',)
+STATICFILES_FINDERS = (
+    'django.contrib.staticfiles.finders.FileSystemFinder',
+    'django.contrib.staticfiles.finders.AppDirectoriesFinder',
+    'compressor.finders.CompressorFinder',
+)
+COMPRESS_URL = "/media/"
+COMPRESS_ROOT = "media/"
+AWS_QUERYSTRING_AUTH = False
```

Comment:
These settings are now all specified in ccnmtlsettings, which it looks like brownfield is using. See: https://github.com/ccnmtl/ccnmtlsettings/blob/master/ccnmtlsettings/shared.py

We should update the documentation on the wiki to make note of this.